src/sql/src/session/vars/definitions.rs

```diff
@@ -1391,7 +1391,7 @@ pub static STATEMENT_LOGGING_TARGET_DATA_RATE: VarDefinition = VarDefinition::ne
 
 pub static STATEMENT_LOGGING_MAX_SAMPLE_RATE: VarDefinition = VarDefinition::new_lazy(
     "statement_logging_max_sample_rate",
-    lazy_value!(Numeric; || 0.0.into()),
+    lazy_value!(Numeric; || 0.99.into()),
```

Comment:
This (and the next one) can cause big problems if you run a lot of queries, but if we want console to have statement logging would be nice to have.

Reply:
@ParkMyCar

Reply:
This LGTM, if folks are having CPU issues on the `mz_catalog_server` we can suggest they reduce these statement logging related params